tests/qam-minimal/install_patterns.pm

```diff
@@ -19,14 +19,14 @@ use testapi;
 sub run {
     select_console 'root-console';
 
-    script_run("while pgrep packagekitd; do pkcon quit; sleep 1; done");
+    script_run("pkcon quit; while pgrep packagekitd; do sleep 1; done");
 
```

Comment:
now that you found out so you have so many duplicates of this - how about a function ? :)